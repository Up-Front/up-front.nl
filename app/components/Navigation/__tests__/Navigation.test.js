import React from 'react'

const ReactShallowRenderer = require('react-test-renderer/shallow');
const renderer = new ReactShallowRenderer();

import { Navigation } from '../index';

it('renders correctly', () => {
  renderer.render(<Navigation />);

  const result = renderer.getRenderOutput();

  expect(result).toMatchSnapshot();
});
