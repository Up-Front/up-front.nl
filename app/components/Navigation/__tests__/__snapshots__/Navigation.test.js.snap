// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className={undefined}
>
  <div>
    <Link
      onlyActiveOnIndex={false}
      style={Object {}}
      to="/"
    >
      Home
    </Link>
    <Link
      onlyActiveOnIndex={false}
      style={Object {}}
      to="/Switches"
    >
      Switches
    </Link>
  </div>
  <div>
    <Connect(StateSync) />
  </div>
</div>
`;
